src/RoleObjectFactory.js

```diff
@@ -135,10 +136,6 @@ function createAccessibilityObjectForRole(config) {
 
       // rest of the roles in alphabetical order by its class entry
 
-    case ROLES.APPLICATION:
```

Comment:
Move to line 159 to actually follow the ordering specified in the comment on line 137